app/src/main/java/com/example/myapplication/file_joiner.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+# Specify the directory path
+directory_path = r'C:\Users\camil\Documents\ScoutingData'
+
```

Comment:
Ensure the directory exists before attempting to list files in it

Reply:
Nice catch, this is something i used since j was the only one using this code but its definetly something that should be modified to be universal